Directory.Build.props

```diff
@@ -62,6 +62,11 @@
     <MachineIndependentBuild>true</MachineIndependentBuild>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <AssemblyInformationCachePaths Condition="Exists('$(MSBuildBinPath)\SdkPrecomputedAssemblyReferences.cache')">$(MSBuildBinPath)\SDKPrecomputedAssemblyReferences.cache</AssemblyInformationCachePaths>
```

Comment:
You shouldn't need this, just the next one.

Reply:
I tried building using `dotnet build MSBuild.sln`, and without this line, AssemblyInformationCachePaths was empty. With it, it had a real value.

Can you clarify why it's unnecessary?

Reply:
There should be no need to have two separate definitions of this, since the SDK root is the same. Perhaps there are better properties to use?